javascript/javascript3/week3/Canvas/script.js

```diff
@@ -0,0 +1,59 @@
+// class creation
+const canvas = document.getElementById("myCanvas");
+
+const circleClass = class circle {
+  constructor(x, y, r, fillColor) {
+    this.x = x;
+    this.y = y;
+    this.r = r;
+    this.fillColor = fillColor;
+  }
+
+  draw() {
+    const ctx = canvas.getContext("2d");
+    ctx.beginPath();
+    ctx.arc(this.x, this.y, this.r, 0, 2 * Math.PI, this.fillColor);
+    ctx.stroke();
+    ctx.fillStyle = this.fillColor;
+    ctx.fill();
+  }
+};
+
+// Random color generator
+function getRandomColor() {
+  function c() {
+    const hex = Math.floor(Math.random() * 256).toString(16);
+    return ("0" + String(hex)).substr(-2);
+  }
+  return "#" + c() + c() + c();
+}
+
+// Random number generator
+function getRandomArbitrary(min, max) {
```

Comment:
getRandomInteger is a better name. Random and Arbitrary means the same thing essentially.